store/store.go

```diff
@@ -1326,7 +1345,62 @@ type PaymentInformation struct {
 	Methods                []*PaymentMethod `json:"methods"`
 }
 
+type storeCustomer struct {
+	LatestTosDate     string `json:"latest_tos_date"`
+	AcceptedTosDate   string `json:"accepted_tos_date"`
+	LatestTosAccepted bool   `json:"latest_tos_accepted"`
+}
+
+// ReadyToBuy returns a bool to show if the user's account has accepted T&Cs and has a payment method registered
+func (s *Store) ReadyToBuy(user *auth.UserState) (bool, error) {
```

Comment:
It looks like `ReadyToBuy` always returns false when err != nil and always true when err == nil. So the bool is actually redundant AFAICT, it could just return an error if `ReadyToBuy()` is false and the client code would do `if err := ReadyToBuy(); err != nil { return err }` or similar.